native/cocos/base/ZipUtils.cpp

```diff
@@ -581,6 +594,39 @@ unsigned char *ZipFile::getFileData(const ccstd::string &fileName, uint32_t *siz
     return buffer;
 }
 
+bool ZipFile::getFileData(const ccstd::string &fileName, uint8_t *buffer, size_t bufferSize) {
+    bool res = false;
```

Comment:
Why need to add this function?
`bool ZipFile::getFileData(const ccstd::string &fileName, ResizableBuffer *buffer)`  could not achieve what you want?
The implementation of these two functions are too similar.

Reply:
This is the most primitive interface. This interface is added because of the abstract class of IFileHandle.

Why do you need the raw read file operation interface?
Because sometimes it is not necessary to read all the contents of the file at one time, because if the file is large, it will consume a lot of resources